operator/controllers/config/constants.go

```diff
@@ -68,6 +68,10 @@ const (
 	DriverVersion   = "2.5.0"
 	OperatorVersion = "2.5.0"
 
+	// Number of replica pods for CSI Sidecar deployment
+	ReplicaCount       = int32(2)
+	TolerationsSeconds = int64(60)
```

Comment:
I think TolerationsSeconds can also be int32, as it would be sufficient for tolerations of even many days, any specific reason for using int64?